modules/EvseV2G/doc.rst

```diff
@@ -0,0 +1,43 @@
+.. _everest_modules_handwritten_EvseV2G:
+
+*******************************************
+EvseV2G
+*******************************************
+
+:ref:`Link <everest_modules_EvseV2G>` to the module's reference.
+This module includes a DIN70121 and ISO15118-2 implementation provided by chargebyte GmbH
+
+Feature List
+============
+
+This document contains feature lists for DIN70121 and ISO15118-2 features, which EvseV2G supports.
+These lists serve as a quick overview of which features are supported.
+
+DIN70121
+--------
+
+===============  ==================
+Feature          Supported
+===============  ==================
+DC               :heavy_check_mark:
+ExternalPayment  :heavy_check_mark:
+===============  ==================
+
+ISO15118-2
+----------
+
+=======================  ==================
+Feature                  Supported
+=======================  ==================
+AC                       :heavy_check_mark:
+DC                       :heavy_check_mark:
+TCP & TLS 1.2            :heavy_check_mark:
+ExternalPayment          :heavy_check_mark:
+Plug&Charge              :heavy_check_mark:
+CertificateInstallation  :heavy_check_mark:
+CertificateUpdate        :heavy_check_mark:
+Pause/Resume             :heavy_check_mark:
+Schedule Renegotation    :heavy_check_mark:
+Smart Charging           :heavy_check_mark:
+Internet Service         :heavy_check_mark:
```

Comment:
Some of these need to be un-checked. It was correct in the initial commit, I believe.

Reply:
Thx for noticing! I changed that back.